sdk/go/common/resource/plugin/provider_plugin.go

```diff
@@ -1155,7 +1155,11 @@ func (p *provider) Invoke(tok tokens.ModuleMember, args resource.PropertyMap) (r
 		return nil, nil, err
 	}
 
-	resp, err := client.Invoke(p.ctx.Request(), &pulumirpc.InvokeRequest{Tok: string(tok), Args: margs})
+	resp, err := client.Invoke(p.ctx.Request(), &pulumirpc.InvokeRequest{
+		Tok:             string(tok),
+		Args:            margs,
+		AcceptResources: true,
```

Comment:
The changes in this file are unrelated to the other changes in the PR, but I wanted to ask about it...

On line 1152, we marshal the properties with `KeepResources: p.acceptResources`, and then on like 1170 below we unmarshal the result properties with `KeepResources:  true`. Is there a reason we're not passing a value for `AcceptResources` when calling `client.Invoke`? Should it be `AcceptResources: true` or `AcceptResources: p.acceptResources`?

Same question applies to the change to `StreamInvoke` below.

Reply:
Seems like they should match. @pgavlin ?

Reply:
They should match, yes. I didn't add this earlier b/c of the uncertainty around invokes accepting and returning resources.

Reply:
> They should match, yes.

So it should be `AcceptResources: p.acceptResources`?

Reply:
I went ahead and changed it to `AcceptResources: p.acceptResources`. I'm going to close this PR and cherry-pick the commit into #5905. We can discuss anything else over there.